src/tests/fix-paths/in/.keboola/manifest.json

```diff
@@ -0,0 +1,40 @@
+{
+  "version": 1,
+  "project": {
+    "id": %%TEST_PROJECT_ID%%,
+    "apiHost": "%%TEST_KBC_STORAGE_API_HOST%%"
+  },
+  "sortBy": "path",
+  "naming": {
+    "branch": "{branch_name}",
+    "config": "{component_type}/{component_id}/{config_name}",
```

Comment:
mozno mi to z predchadzajucich PRs uniklo ale co sa stane ak bude naming obsahovat aj config-id tj napr:
`config": "{component_type}/{component_id}/{config_id}-{config_name}",`
fix-paths si potom tiez cez api vygeneruje IDcka a doplni?

Reply:
ano, iba v testoch je to vypnute, kedze IDcka su nahodne a blbo by sa to kontrolovalo.